packages/storage-ui/src/Components/Modules/MoveFileModal/MoveFileModal.tsx

```diff
@@ -187,29 +212,48 @@ const MoveFileModule = ({ filesToMove, modalOpen, onClose, onCancel, mode }: IMo
       <Grid
         item
         flexDirection="row"
-        justifyContent="flex-end"
+        justifyContent="space-between"
+        alignItems="center"
         className={classes.paddedContainer}
       >
-        <CustomButton
-          onClick={onCancel}
-          size="medium"
-          className={classes.cancelButton}
-          variant={desktop ? "outline" : "gray"}
-          type="button"
-        >
-          <Trans>Cancel</Trans>
-        </CustomButton>
-        <Button
-          variant="primary"
-          size={desktop ? "medium" : "large"}
-          type="submit"
-          className={classes.okButton}
-          loading={movingFile}
-          disabled={!movePath}
-          onClick={onMoveFile}
+        {!!movePath && !isAllowedToMove && (
+          <Typography
+            component="p"
+            variant="body1"
+          >
+            {
+              folders.length
```

Comment:
The error coming up wrong,
```suggestion
              movePath !== currentPath
```